### ISSUE TITLE
[Skype] Bot nonresponsive over Skype, works fine in web chat

### ISSUE THREAD

**Issue by muppetaphrodite** [Microsoft/BotBuilder#3488]
## Bot Info 
* Bot handle: HereToHelp-App
* App ID: 1fd57773-ddff-4844-8896-e8b0019cab35
* SDK Platform: Node
* SDK Version: 3.9.1
* Active Channels: Web, Slack (nonresponsive)
* Deployment Environment: Azure Bot Service, App Service Plan, Continuous Deployment VSTS

## Issue Description
This bot is nonresponsive over Skype, even though it works fine both locally and on web chat channels. I have a similar bot working well on a consumption plan in both Web and Skype. The only main difference is the state storage - are there perhaps Skype-specific environment keys that didn't get added when the channel was activated?

## Reproduction Steps
1. Send a query to bot over Web Chat (i.e. "how are you?") and same to Skype bot

## Expected Behavior
Bot responds to queries ("hello", "how are you" "help", etc...) identically over both channels.

## Actual Results
Web Chat works fine. Skype seems to be unresponsive and the server logs aren't showing any errors.

## Code Example
```
var tableName = 'BotStore';
var storageName = 'heretohelpapp';
var storageKey = process.env.AZURE_STORAGE_KEY;
var azureTableClient = new azure.AzureTableClient(tableName, storageName, storageKey);
var tableStorage = new azure.AzureBotStorage({ gzipData: false }, azureTableClient);

// Setup Restify Server
var server = restify.createServer();
server.listen(process.env.port || process.env.PORT || 3978, function () {
   console.log('%s listening to %s', server.name, server.url); 
});
  
// Create chat connector for communicating with the Bot Framework Service
var connector = new builder.ChatConnector({
    appId: process.env.MicrosoftAppId,
    appPassword: process.env.MicrosoftAppPassword,
    stateEndpoint: process.env.BotStateEndpoint,
    openIdMetadata: process.env.BotOpenIdMetadata 
});

// Listen for messages from users 
server.post('/api/messages', connector.listen());

var bot = new builder.UniversalBot(connector, function (session) {
    console.log("Initialized the bot.");
    session.send("prompt_Chat_Intro");
}).set('storage', tableStorage);

// Add global LUIS recognizer to bot
<insert LUIS model here>

<insert chat logic here>
```

**Comment by stevengum97** [comment]
@muppetaphrodite, which ABS template did you use for this bot? Was it the LUIS template?

**Comment by muppetaphrodite** [comment]
For this one, I started with a mirror bot and moved over the code I needed from my working Consumption version of the bot (pointed at the same LUIS model).

**Comment by stevengum97** [comment]
Have you tried commenting out `stateEndpoint` and `openIdMetadata` in your ChatConnector settings?

**Comment by muppetaphrodite** [comment]
Funny you should mention that. They were originally commented out when I discovered this problem; I un-commented them in case that was somehow required. And YET. Re-commenting them seems to have unblocked the bot, so I'm thankful for the idea regardless.

**Comment by stevengum97** [comment]
Awesome, I'm glad it worked! 😃 
___

For future reference:

So I tested some items, and here's what I found in regards to the `ChatConnector`'s instantiation:
```js
var connector = new builder.ChatConnector({
    appId: process.env.MicrosoftAppId,
    appPassword: process.env.MicrosoftAppPassword,
    stateEndpoint: process.env.BotStateEndpoint,
    openIdMetadata: process.env.BotOpenIdMetadata 
});
```
`BotStateEndpoint` and `BotOpenIdMetadata` doesn't exist in the Application Settings. Not there isn't a value associated with `BotStateEndpoint` and `BotOpenIdMetadata`, but the keys themselves don't exist. 

Either removing those configuration options from your ChatConnector:
```js
var connector = new builder.ChatConnector({
    appId: process.env.MicrosoftAppId,
    appPassword: process.env.MicrosoftAppPassword
});
```
Or adding the keys to my Application Settings fixed the issue for me. 

![3488](https://user-images.githubusercontent.com/14935595/30618918-75d96622-9d51-11e7-88a0-0800b28349e1.PNG)
